packages/eslint/src/generators/convert-to-flat-config/__snapshots__/generator.spec.ts.snap

```diff
@@ -317,7 +317,49 @@ module.exports = [
 "
 `;
 
-exports[`convert-to-flat-config generator should handle custom eslintignores 1`] = `
+exports[`convert-to-flat-config generator should convert json successfully 1`] = `
+"const { FlatCompat } = require('@eslint/eslintrc');
+const nxEslintPlugin = require('@nx/eslint-plugin');
+const js = require('@eslint/js');
```

Comment:
We'd previously discussed improving the newline spacing here in the converted output